src/main/kotlin/com/coder/gateway/views/steps/CoderWorkspaceProjectIDEStepView.kt

```diff
@@ -54,6 +56,7 @@ import com.jetbrains.gateway.ssh.IdeWithStatus
 import com.jetbrains.gateway.ssh.IntelliJPlatformProduct
 import com.jetbrains.gateway.ssh.deploy.DeployException
 import com.jetbrains.gateway.ssh.util.validateRemotePath
+import com.jetbrains.rd.generator.nova.PredefinedType
```

Comment:
You probably already see the warning but appears to be unused (I think `ktlint` will delete unused imports if you want to try using that)